core/src/main/scala/chisel3/properties/Path.scala

```diff
@@ -27,33 +36,41 @@ object Path {
 
   /** Construct a Path that refers to a Module
     */
-  def apply(module: BaseModule): Path = {
+  def apply(module: BaseModule): Path = apply(module, false)
+  def apply(module: BaseModule, _isMemberTarget: Boolean): Path = {
     new Path {
-      def toTarget(): IsMember = module.toAbsoluteTarget
+      def toTarget():     IsMember = module.toAbsoluteTarget
+      def isMemberTarget: Boolean = _isMemberTarget
     }
   }
 
   /** Construct a Path that refers to a Data
     */
-  def apply(data: Data): Path = {
+  def apply(data: Data): Path = apply(data, false)
+  def apply(data: Data, _isMemberTarget: Boolean): Path = {
     new Path {
```

Comment:
```suggestion
  def apply(data: Data, isMemberTarget: Boolean): Path = {
    val _isMemberTarget = isMemberTarget // avoid name shadowing below
    new Path {
```
Since this is a public API, it should not have an `_`.

Reply:
👍 thanks for pointing it out, I forgot this was a public API.